.gitignore

```diff
@@ -1,74 +1,16 @@
-*.iml
 .gradle
 /local.properties
 /.idea/workspace.xml
-/.idea/libraries
 .DS_Store
-/build
-/captures
-.externalNativeBuild
-.idea
-/*.iml
-
-# files for the dex VM
-*.dex
-
-# Java class files
-*.class
-
-# generated files
-bin/
-gen/
-out/
+.idea/
 build/
-.externalNativeBuild
-.cxx
-iosApp/Podfile.lock
-iosApp/Pods/*
-iosApp/iosApp.xcworkspace/*
-iosApp/iosApp.xcodeproj/*
-!iosApp/iosApp.xcodeproj/project.pbxproj
-mifospay-shared/mifospay-shared.podspec
-
-# Eclipse project files
-.classpath
-.project
-
-# Windows thumbnail db
-.DS_Store
-
-# IDEA/Android Studio project files, because
-# the project can be imported from settings.gradle.kts
-*.iml
-.idea/*
-!.idea/copyright
-# Keep the code styles.
-!/.idea/codeStyles
-/.idea/codeStyles/*
-!/.idea/codeStyles/Project.xml
-!/.idea/codeStyles/codeStyleConfig.xml
-
-# Kotlin
-.kotlin
-
-# Android Studio captures folder
-captures/
-
-/app/app-release.apk
-app/app.iml
-app/manifest-merger-release-report.txt
-
-# Exclude Google services from prod flavour
-androidApp/src/prod/google-services.json
-
-#*.keystore
-
-version.txt
-fastlane/report.xml
-firebaseAppDistributionServiceCredentialsFile.json
```

Comment:
do not remove some of these mentioned from gitignore for ex: `firebaseAppDistributionServiceCredentialsFile.json`. have a look at the kmp-project-template what are there if it is there then keep it also here and the rest which u think might not be needed then delete.